blueprints/app/files/package.json

```diff
@@ -42,6 +42,7 @@
     "ember-source": "~3.5.1<% if (welcome) { %>",
     "ember-welcome-page": "^3.2.0<% } %>",
     "eslint-plugin-ember": "^5.2.0",
+    "eslint-plugin-node": "^7.0.1",
```

Comment:
should be updated to latest version (currently 8.0.1)

Reply:
v7 is already brought in via the addon. I wouldn't want to touch the version in this PR. Created https://github.com/ember-cli/ember-cli/pull/8572.

Reply:
> I wouldn't want to touch the version in this PR

Why?